### ISSUE TITLE
Update Besley

### ISSUE THREAD

**Issue by EwonRael** [google/fonts#4768]
**Font Project Git Repo URL:** https://github.com/indestructible-type/Besley

**Current version:** 2.0

**New available version:** 3.0

**Super short description of the upgrade:**
more glyphs, new condensed and narrow widths

**Comment by emmamarichal** [comment]
Hi @EwonRael!
What is the status here? :)

**Comment by SapphireSlash** [comment]
I think the least y’all can do is upgrade Besley to 2.01 because that's the latest release on the Besley repository. Also, 2.01 adds small caps while the Besley on Google Fonts (_which is 2.0_) doesn't have them.

**Comment by thlinard** [comment]
> I think the least y’all can do is upgrade Besley to 2.01 because that's the latest release on the Besley repository. Also, 2.01 adds small caps while the Besley on Google Fonts (_which is 2.0_) doesn't have them.

Version 4 is available: https://github.com/indestructible-type/Besley/tree/master/fonts


> Changelog
> 
> v1.0 Original Release.
> v1.1 Added the Fatface weight. Corrected kerning.
> v2.0 Redrew letterforms to enable variable font versions. Improved language support. Added options for old-style and tabular numbers. Improved kerning and various other improvements.
> v2.01 Added small caps designed by [Fredrick Brennan](https://github.com/ctrlcctrlv/)
> v3.0 Added a Narrow and Condesed version of the font. Added Cyrillic. Added dingbats.
> 4.0 Added support for the Greek Language

**Comment by SapphireSlash** [comment]
> I don't see a release since early 2022.  Keeping this in the Icebox

@thlinard 4.0 does not have a release on the repo, 2.0.1 is the latest release on the repo.

**Comment by thlinard** [comment]
> > I don't see a release since early 2022.  Keeping this in the Icebox
> 
> [@thlinard](https://github.com/thlinard) 4.0 does not have a release on the repo, 2.0.1 is the latest release on the repo.

No, there is no "Release" **tag** after version 2.01, but the [commit comments](https://github.com/indestructible-type/Besley/commits/master/) clearly indicate that version 4 is present in https://github.com/indestructible-type/Besley/tree/master/fonts
